test/behaviour/connection/ConnectionStepsCluster.java

```diff
@@ -45,6 +47,7 @@ void beforeAll() {
         }
         server.start();
         TypeDBSingleton.setTypeDBRunner(server);
+        ConnectionStepsBase.isBeforeAllRan = true;
```

Comment:
This isn't necessary, right? Since the implementation of `before` in `ConnectionStepsBase` already sets `isBeforeAllRan` to `true`,

(having said that, `isBeforeAllRan` being set in a method that isn't `beforeAll` feels pretty dodgy, so perhaps there is a better solution here)

Reply:
cleaned it up

Reply:
Urgh actually im going to revert this, and when we upgrade Cucumber we can use a new version that has `@BeforeAll` and `@AfterAll`